moxin-frontend/src/shared/widgets/c_button.rs

```diff
@@ -0,0 +1,92 @@
+use makepad_widgets::*;
+
+live_design!(
+    import makepad_widgets::base::*;
+    import makepad_widgets::theme_desktop_dark::*;
+    import crate::shared::styles::*;
+    import crate::shared::widgets::*;
+
+    CButton = {{CButton}} <RoundedView> {
+        align: {x: 0.5, y: 0.5}
+        flow: Right
+        width: Fit, height: Fit
+        padding: { top: 15, bottom: 15, left: 8, right: 13}
+        spacing: 10
+        draw_bg: {
+            radius: 2.0,
+            color: #fff,
+            border_width: 1.0,
+            border_color: #000,
+        }
+
+        icon = <Icon> {
+            draw_icon: {
+                fn get_color(self) -> vec4 {
+                    return #000;
+                }
+            }
+            icon_walk: {width: 12, height: 12}
+        }
+
+        label = <Label> {}
+    }
+);
+
+/// An attempt of a custom button, sharable with the rest of the app.
+#[derive(Widget, Live, LiveHook)]
+pub struct CButton {
```

Comment:
Also, we are using views as buttons everywhere in the app, i think we should make this change in another pr, covering every case.

Reply:
Covered in slack discussion